tests/runtime/config

```diff
@@ -25,3 +25,13 @@ RUN {{BPFTRACE}} -e 'config = { debug_output=1 } uprobe:./testprogs/uprobe_test:
 EXPECT stdin:1:12-25: ERROR: debug_output can only be set as an environment variable
 BEFORE ./testprogs/uprobe_test
 WILL_FAIL
+
+NAME maps are printed by default
+RUN {{BPFTRACE}} -e build/src/bpftrace -e 'uprobe:./testprogs/uprobe_test:uprobeFunction1 { @[comm] = count(); exit(); }' -p {{BEFORE_PID}}
+EXPECT @[uprobe_test]: 1
+BEFORE ./testprogs/uprobe_test
```

Comment:
nit: simplify to:
```suggestion
PROG BEGIN { @[comm] = count(); exit(); }'
EXPECT @[bpftrace]: 1
```
uprobe is not necessary, so using BEGIN will help make the uprobe stuff less load bearing.

same for below

Reply:
Thanks! Much cleaner, I will update shortly